sdk/core/core-client/review/core-client.api.md

```diff
@@ -297,6 +294,9 @@ export interface PolymorphicDiscriminator {
 // @public
 export type QueryCollectionFormat = "CSV" | "SSV" | "TSV" | "Pipes" | "Multi";
 
+// @public
+export type RawResponseCallback = (response: OperationResponse, rawResponse: FullOperationResponse) => unknown;
```

Comment:
Can you put the `rawResponse` as the first parameter, since that's probably what they're after, and that way they don't have to ignore the regular response if they don't need it?

Also I'm on the the fence if we need to include the regular response or not. On one hand it seems nice for completeness, but on the other hand they could just use what comes back from the async method.